packages/babel-helper-create-class-features-plugin/src/fields.ts

```diff
@@ -557,14 +557,17 @@ function buildPrivateFieldInitLoose(
   const { id } = privateNamesMap.get(prop.node.key.id.name);
   const value = prop.node.value || prop.scope.buildUndefinedNode();
 
-  return template.statement.ast`
+  return inheritPropComments(
+    template.statement.ast`
     Object.defineProperty(${ref}, ${t.cloneNode(id)}, {
```

Comment:
Nit: can you indent the template contents one level more, to be visually inside the template tag "call"? In various places in this file.